docs/1.getting-started/3.configuration.md

```diff
@@ -133,3 +133,56 @@ Name                                          | Config File             | How To
 | [Stylelint](https://stylelint.io)           | `.stylelintrc.json`     | [More Info](https://stylelint.io/user-guide/configure)
 | [TailwindCSS](https://tailwindcss.com)      |  `tailwind.config.js`   | [More Info](https://tailwindcss.nuxtjs.org/tailwind/config/)
 | [Vitest](https://vitest.dev)                | `vitest.config.ts`      | [More Info](https://vitest.dev/config/)
+
+## Vue Plugin Configuration
+
+If you need to pass some options to `@vitejs/plugin-vue` or `@vitejs/plugin-vue-jsx`, you should also do it using `nuxt.config.ts`. For this purpose, these parameters are present in the Vite configuration:
+- `vite.vue` for `@vitejs/plugin-vue`. Check available options [here](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue).
+- `vite.vueJsx` for `@vitejs/plugin-vue-jsx`. Check available options [here](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue-jsx).
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  vite: {
+    vue: {
+      customElement: true
+    },
+    vueJsx: {
+      mergeProps: true
+    }
+  }
+})
+```
+
+### Enabling Experimental Vue Features
+
+You may need it to enable experimental features in Vue, e.g. `defineModel` or `propsDestructure`. To do this you need to add this to your `nuxt.config.ts`:
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  vite: {
+    vue: {
+      script: {
+        defineModel: true,
+        propsDestructure: true
+      }
+    }
+  }
```

Comment:
Once we update the `vue` schema types (https://github.com/nuxt/nuxt/pull/21364#discussion_r1217194604) this would probably be a bit better:

```suggestion
  vue: {
    script: {
      defineModel: true,
      propsDestructure: true
    }
  }
```

Reply:
@danielroe do you want to keep inside the script object?

Reply:
Ah, no, you're right - they top level seems good to me. (vue-loader options don't have script object so 🤷‍♂️)

Reply:
So maybe I should create 3 separate sections?

- Vue experimental features
- How to configure vite-plugin-vue
- How to configure vue-loading

Reply:
Updated in https://github.com/nuxt/nuxt/pull/21303/commits/90c9cbfb1bcd3849bab11a52f7c2296283a64eb8.